lms/templates/courseware/progress.html

```diff
@@ -48,7 +48,7 @@
             <h1 class="progress-certificates-title">${_("Course Progress for Student '{username}' ({email})").format(username=student.username, email=student.email) | h}</h1>
           </header>
 
-          %if show_generate_cert_btn:
+          %if show_generate_cert_btn and not is_invalid_cert:
```

Comment:
If we have `is_invalid` check within the `cert_generation_enabled` function, we won't need this double negative check.